### DISCUSSION TITLE
SP-API Issue - App ID: amzn1.sp.solution.34602ec6-8f08-4829-a96b-a99c9c5688e3 Error Code: MD9001

### DISCUSSION THREAD

**Discussion by nrogers805** [amzn/selling-partner-api-models#3503] [General]
This error started this morning.

The SP-API workflow was working fine until today.

I tried updating the App from Hybrid to SP-API but it did not help as the error is still occurring 

Any ideas?

It seems many other developers, with Hybrid Apps,  are having same issue

**Comment by sungolivia** [comment]
any other error messages?

**Reply by nrogers805** [reply]
> any other error messages?

No just that message ? odd question though ?

**Comment by sungolivia** [comment]
#2916

**Reply by nrogers805** [reply]
> #2916

yes thanks, already contributing to that thread
